components/clp-py-utils/clp_py_utils/initialize-orchestration-db.py

```diff
@@ -55,6 +55,7 @@ def main(argv):
                     `status_msg` VARCHAR(512) NOT NULL DEFAULT '',
                     `creation_time` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
                     `start_time` DATETIME(3) NULL DEFAULT NULL,
+                    `update_time` DATETIME(3) NULL DEFAULT NULL,
```

Comment:
Shall we also set `DEFAULT CURRENT_TIMESTAMP(3)`?

Reply:
Depends? the creator of this row should be webui? so if the update_time = null means scheduler never updates it, which would make more sense.

Reply:
> the creator of this row should be webui?

only the scheduler updates it

webui queries all jobs based on a comparison on this column. i'm not too sure about the behaviour of comparing something against NULL. if the rows who have NULL `update_time` cannot be included, we should probably avoid assigning NULL here or include NULL in the WHERE clause.

On the other hand, for consistency, I believe it's good to always assign a value to the column. Its value should match `creation_time` at the job's initialization.

Reply:
Sure, it might be a good idea for avoiding NULL.

The issue I wanted to point out is that, if we keep the default Null, we can then catch all jobs that are created by webui but never reached to scheduler. But alternatively, we can also do SELECT WHERE last_update_time = creation_time.